android_jni/avifandroidjni/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
maybe add a newline to this and a few of the other files to avoid any unrelated diffs in the future. vim should add an eol automatically if you write the file; though if they're autogenerated then it's probbably fine to leave them.

Reply:
Yeah this was auto-generated by Android Studio.